code/style.css

```diff
@@ -1,5 +1,635 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+* {
+  /*** outline: 1px solid red;***/
+  box-sizing: border-box;
+}
+
+/*** ALT BACKGROUND COLOR body{
+  background: rgb(255,255,255);
+  background: radial-gradient(circle, rgba(255,255,255,1) 0%, rgba(148,187,233,1) 100%);
+  margin: 0px;
+}
+***/
 
 body {
-  background-color: peachpuff;
+  background-color: #f3bbff;
+  animation: bgColor 50s infinite linear;
+  margin: 0px;
+}
+
+@media screen and (max-width: 667px) {
+  body {
+    margin: 0 auto;
+    width: 100%;
+  }
+}
+
+@keyframes bgColor {
+  10% {
+    background-color: #a43535;
+  }
+  20% {
+    background-color: #ff7b00;
+  }
+  30% {
+    background-color: #ffff51;
+  }
+  40% {
+    background-color: #a7ff4e;
+  }
+  50% {
+    background-color: #7addfe;
+  }
+  60% {
+    background-color: #6b6bfd;
+  }
+  70% {
+    background-color: #ca61ff;
+  }
+  80% {
+    background-color: #ff54af;
+  }
+  90% {
+    background-color: #f3bbff;
+  }
+}
+
+h1 {
+  font-family: 'Fira Sans', sans-serif;
+  font-size: 100px;
+  margin: 0px;
+  font-weight: 400;
+}
+
+@media screen and (max-width: 667px) {
+  h1 {
+    font-size: 30px;
+  }
+}
+
+h2 {
+  font-family: 'Fira Sans', sans-serif;
+  font-size: 50px;
+  font-style: italic;
+  font-weight: 400;
+}
+
+@media screen and (max-width: 667px) {
+  h2 {
+    font-size: 50px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  h2 {
+    font-size: 80px;
+  }
+}
+
+h3 {
+  font-family: 'EB Garamond', serif;
+  font-size: 30px;
+}
+
+h4 {
+  font-family: 'Fira Sans', sans-serif;
+  font-size: 40px;
+  font-weight: 200;
+}
+
+@media screen and (min-width: 1024px) {
+  h4 {
+    font-size: 60px;
+  }
+}
+
+@media screen and (max-width: 667px) {
+  h4 {
+    font-size: 25px;
+  }
+}
```

Comment:
You wrote that you struggled with the DRY principle so maybe you're already aware of this - but a good rule of thumb is to keep the media queries in order. If you're styling desktop first, then you can style desktop first, then tablet, and lastly mobile (or the other way around). Also, you can have many rules for many elements inside one media query. Something like this:

```
/*Default (desktop in your case)*/
h1 {
  /*styling for default*/
}

h2 {
  /*styling for default*/
}

h3 {
  /*styling for default*/
}

h4 {
  /*styling for default*/
}

/*Tablet*/
@media screen and (max-width: 667px) {
  h1 {
    /*styling for tablet*/
  }

  h2 {
    /*styling for tablet*/
  }

  h3 {
    /*styling for tablet*/
  }

  h4 {
    /*styling for tablet*/
  }
}
```
and so on